pkg/github/github.go

```diff
@@ -18,6 +20,7 @@ type Githuber interface {
 	Teams(org string) ([]*Team, error)
 	Repositories(org string) ([]*Repository, error)
 	CreateDeployKey(org, repository, title, publicKey string) (*Key, error)
+	ListTeamMembers(team client.GithubTeam) ([]client.GithubTeamMember, error)
```

Comment:
Is there any reason it's called ListTeamMembers here, vs GetGithubTeamMembers above (list vs get)?

Reply:
Hmm, well - it takes a team as an argument, and does not really "get" anything, just some putting things in correct objetcs and returning.
Also, underlying command in github package is called ListTeamMembersByID